cmd/plural/cd.go

```diff
@@ -93,6 +93,17 @@ func (p *Plural) doInstallOperator(url, token string) error {
 	if err != nil && !apierrors.IsAlreadyExists(err) {
 		return err
 	}
+	consoleClient, err := console.NewConsoleClient(token, url)
+	if err != nil {
+		return err
+	}
+	myCluster, err := consoleClient.MyCluster()
+	if err != nil {
+		return err
+	}
+	if !confirm(fmt.Sprintf("Are you sure you want to install deploy operator for %s cluster?", myCluster.MyCluster.Name), "PLURAL_INSTALL_AGENT_CONFIRM") {
```

Comment:
Could we perhaps print a more thorough description of the cluster, eg something close to its describe:

```
Name: <name>
Handle: <handle>
Provider: (if it exists)
```

That way users have a really transparent view of what the cluster is.

Reply:
![image](https://github.com/pluralsh/plural-cli/assets/13030040/dd2d4e0e-bcd0-45a1-94cc-5fd419feee56)
done